features/product/viewing_products/viewing_different_price_for_different_product_variants.feature

```diff
@@ -0,0 +1,22 @@
+@viewing_products
+Feature: Viewing different price for different product variants
+    In order to see product variant price
+    As a Visitor
+    I want to be able to see proper price for each product variant
+
+    Background:
+        Given the store operates on a single channel in "United States"
+        And the store has a "Wyborowa Vodka" configurable product
+        And the product "Wyborowa Vodka" has "Wyborowa Vodka Exquisite" variant priced at "$40.00"
+        And the product "Wyborowa Vodka" has "Wyborowa Apple" variant priced at "$4.00"
```

Comment:
Their prices differ by just one zero, can we use some other numbers than 4 and 0 to make the difference more clear?